en/console-guide.md

```diff
@@ -2,52 +2,52 @@
 
 ## Manage flow logs
 ### Create Flow Log
-It can be created through the Create flow log in the NHN Cloud console. For flow logs to work correctly, the flow log system must have access to storage. For more information, see **Granting storage access for flow logs**below.
+It can be created through the Create flow log in the NHN Cloud console. For flow logs to work correctly, the flow log system must have access to storage. For more information, see **Granting storage access for flow logs** below.
 
 
-After you click the **Create flow log** button, you can set basic information about the flow log. The settings require the following items.
+After you click the **Create Flow Log** button, you can set basic information about the flow log. The settings require the following items.
 
 
 * Name: Enter a name for the flow log.
 * Description: Describe the flow log.
-* Collect target: Allows you to select the flow log collection unit. Currently, you can only create `network interface` units. Here you can see a list of network interfaces that exist in your project, and select the ones you want to collect.
-* Collect traffic type: You can select **Allow**, **Block**, or both. If you select Allow, only packets allowed to **Security Groups** are collected; if you click Block, only blocked packets are collected. If you select both, both allowed and blocked packets are collected and aggregated.
+* Collect Target: Allow you to select the flow log collection unit. Currently, you can only create `network interface` units. Here you can see a list of network interfaces that exist in your project, and select the one you want to collect.
+* Type of Traffic Collected: You can select **Allow**, **Block**, or both. If you select Allow, only packets allowed to **Security Groups** are collected; if you click Block, only blocked packets are collected. If you select both, both allowed and blocked packets are collected and aggregated.
 
 * Collect only connection attempt packets: If checked, packets after the connection is established are not collected. For TCP, packets with the TCP state established are not collected, and for UDP/ICMP, response packets are not collected.
-* Collection interval: The time interval to clean and create statistics through collected packets and create a file in storage. It can be set between 1 minute and 15 minutes. 
-* File storage path: Currently, only Object Storage is supported. For **Object Storage**, enter the OBS endpoint, AUTH_tenant, container, and path at once.
+* Collection Interval: The time interval to aggregate and create statistics through collected packets and create a file in storage. It can be set between 1 minute and 15 minutes. 

```

Comment:
```suggestion
* Collection Interval: The time interval to sort the collected packets, generate statistics, and create a file in storage. It can be set between 1 minute and 15 minutes. 
```